packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -50,7 +50,7 @@ const DEFAULT_DECIMALS = 9;
  * added in the appropriate place and settled when the price reaches that level.
  */
 
-const trace = makeTracer('AucBook', false);
+const trace = makeTracer('AucBook', true);
```

Comment:
did we get the wear-leveling option on these `makeTracer` bits? :)

Reply:
yeah it was part of the durability upgrade